client/cli/ok.py

```diff
@@ -92,6 +93,12 @@ def parse_input(command_input=None):
 
 def main():
     """Run all relevant aspects of ok.py."""
+
+    #Checking user's Python bit version
+    bit_v = (8 * struct.calcsize("P"))
+    if (bit_v == 32):
+        print("32 bit Python is not supported. Please install the 64 bit version.")
```

Comment:
This is especially concerning when 32bit errors with SSL result in tracebacks (see #152). The traceback itself will consume the screen. I don't think we should automatically give up with `exit(0)` just because there is an auth error. 

I propose the following: 
- Always log whether the client is 32bit or 64bit 
- Except the SSL errors and auth errors - if those error are raised while running the protocols, except them and ensure the bit version is 64 bit. If it's not - print a string about installing 64 bit

This effectively combines #152 and this PR. If you're ok with that course of action, Hayden's PR should just be limited to logging the bit version. 

This would mean that we'd still allow students to run it locally and have software updates etc.